### PR TITLE
Add translator assertions

### PR DESCRIPTION
**Pull request by moiristo**
Part of #1618

- Originally included in BE, but useful in other projects as well
- Bumped rails

### CODE REVIEW

**Comment by ghost** [comment]
@moiristo, thanks for your PR! By analyzing the history of the files in this pull request, we identified @everplays and @ruuts to be potential reviewers.

File: lib/translator/assertions.rb [hunk 1]
```diff
@@ -0,0 +1,80 @@
+module Translator
+  module Assertions
+    extend ActiveSupport::Concern
```
Comment (ruuts) [inline lib/translator/assertions.rb:3]: why not make this part of our toolchain?
it's now being copied over many projects.

**Review by ruuts — COMMENTED** [review]
He man this can be merged but I think it's better that you move this to the toolchain and from their use it in all of our projects. What do you think?

**Comment by moiristo** [comment]
@ruuts going to merge this as discussed, because the assertions are dependent on `Translator`.

**Pull request merged**